tensorflow-framework/src/main/java/org/tensorflow/framework/regularizers/L1L2.java

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.regularizers;
+
+import org.tensorflow.Operand;
+import org.tensorflow.framework.losses.impl.LossesHelper;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.family.TNumber;
+
+/**
+ * A regularizer that applies both L1 and L2 regularization penalties.
+ *
+ * <p>The L1 regularization penalty is computed as:
+ *
+ * <pre>loss = l1 * reduceSum(abs(x))</pre>
+ *
+ * <p>The L2 regularization penalty is computed as
+ *
+ * <pre>loss = l2 * reduceSum(square(x))</pre>
+ *
+ * <p>The difference between this class and the {@link L1_L2} is use of the default regularization
```

Comment:
Actually it looks like `l1_l2` was the after thought in Keras, as it is defined as a method. I am ok with adding the `l1_l2` defaults to `L1L2` and perhaps add a static method to `L1L2` to create an L1L2  class with the current `l1_l2` defaults.

Perhaps something like:

```
public static L1L2    l1_l2() {
    return new L1L2(DEFAULT_REGULARIZATION_PENALTY, DEFAULT_REGULARIZATION_PENALTY);
}
```

Reply:
That seems fine to me.

Reply:
`create` or `createDefault`? Alternatively, I could change the constructors that do not specify the l1, l2 values,

```
 public L1L2(Ops tf) {
    this(tf, DEFAULT_REGULARIZATION_PENALTY, DEFAULT_REGULARIZATION_PENALTY);
  }
```

Reply:
I *would* lean toward changing the no-specified-penalty constructors to use the defaults instead of `0.f`. But if factory methods felt more comfortable to you, and if you felt that fit some overall direction of the code base, I could be comfortable with that.

Yeah, I started down the `createDefault` path, but realized the idiomatic pattern is just overloaded `create` methods.

Reply:
I am changing `L1L2(Ops tf)` to use t `DEFAULT_REGULARIZATION_PENALTY` and eliminating class `L1_L2`.

Reply:
👍 done